### ISSUE TITLE
Style/OpMethod should not complain about the backticks method

### ISSUE THREAD

**Issue by donv** [bbatsov/rubocop#3320]
Given the following code:

```ruby
def `(cmd)
  system cmd
end
```

Rubocop complains with the following offence:

```
my_class.rb:40:7: C: Style/OpMethod: When defining the  operator, name its argument other`. (https://github.com/bbatsov/ruby-style-guide#other-arg)
def `(cmd)
      ^^^
```

I think that "other" is not an intuitive parameter name for the backticks method and that Rubocop should not report it as an offence.

```
$ rubocop -V
0.41.2 (using Parser 2.3.1.2, running on jruby 2.3.0 java)
```

**Comment by donv** [comment]
@Drenmi @bbatsov Thanks!
